libraries/AP_AHRS/AP_AHRS_DCM.cpp

```diff
@@ -792,7 +792,7 @@ AP_AHRS_DCM::drift_correction(float deltat)
     // flat, but still allow for yaw correction using the
     // accelerometers at high roll angles as long as we have a GPS
     if (AP_AHRS_DCM::use_compass()) {
-        if (have_gps() && is_equal(gps_gain,1.0f)) {
+        if (have_gps() && is_equal<float>(gps_gain,1.0f)) {
```

Comment:
Okay, then it was an AP_Param, and I was enforcing usage of a float function, which is fine ..